<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Markdown Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1064</width>
      <height>602</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayout" stretch="1,1">
     <item>
      <widget class="QPlainTextEdit" name="mdEdit">
       <property name="plainText">
        <string>#欢迎使用SimpleQtMarkdown

##这是二级标题

这是无序列表

- 第一项 
- 第二项
- 第三项

这是分割线

------

这是有序列表

1. 第一项 
2. 第二项
3. 第三项
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWebView" name="view">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFile"/>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="saveAs"/>
    <addaction name="separator"/>
    <addaction name="saveAsHTML"/>
    <addaction name="separator"/>
    <addaction name="exitTheEditor"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="styleMenu">
     <property name="title">
      <string>样式表</string>
     </property>
    </widget>
    <addaction name="undo"/>
    <addaction name="redo"/>
    <addaction name="separator"/>
    <addaction name="styleMenu"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="MarkdownSyntax"/>
    <addaction name="separator"/>
    <addaction name="about"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="newFile">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="saveAsHTML">
   <property name="text">
    <string>导出为HTML</string>
   </property>
   <property name="toolTip">
    <string>保存为HTML文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="exitTheEditor">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="undo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redo">
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="MarkdownSyntax">
   <property name="text">
    <string>Markdown 语法</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exitTheEditor</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>undo</sender>
   <signal>triggered()</signal>
   <receiver>mdEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redo</sender>
   <signal>triggered()</signal>
   <receiver>mdEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAs</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAsHTML</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportAsHtml()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newFile()</slot>
  <slot>save()</slot>
  <slot>saveAs()</slot>
  <slot>exportAsHtml()</slot>
  <slot>open()</slot>
 </slots>
</ui>
